src/services/realtime/base/index.ts

```diff
@@ -60,6 +62,10 @@ export class RealtimeService implements DefaultRealtimeService {
     this.presenceMouseObserver = new Observer({ logger: this.logger });
     this.presenceMouseParticipantLeaveObserver = new Observer({ logger: this.logger });
     this.presenceMouseParticipantJoinedObserver = new Observer({ logger: this.logger });
+    this.presenceSlotsInfosObserver = new Observer({ logger: this.logger });
```

Comment:
The slot is inside the participant, it's a key called slotIndex, you don't need a separated channel, or observer only for this. You can use the participants observers.
```suggestion

```